### ISSUE TITLE
Emote specific chat improvements

### ISSUE THREAD

**Issue by Mefu** [vinszent/gnome-twitch#190]
I have 3 emote related chat improvement suggestions. I provided screenshots from BTTV chat using firefox.

First is tooltips when hovering over emotes. It is useful to see what you need to write to get a specific emote when you first see it.

Tooltip from bttv
![tooltip from bttv](http://i.imgur.com/kWVN22s.png)

Tooltip for channel emotes
![channel specific tooltip](http://i.imgur.com/Nj3TvoL.png)

Second one, when you select text that contains emotes and copy it, you should get text versions of emotes to clipboard. 

![copy paste](http://i.imgur.com/uAu0VVf.png)

Last one, when you type and press Tab (or another key for autocomplete, maybe make it customizable too), you should get your text autocompleted to closest emote text.

Autocomplete - to get this, I wrote "kapp" and pressed Tab, it autocompleted to Kappa and showed other longer options also. I can cycle through options using arrow keys or simply select with mouse.
![auto complete](http://i.imgur.com/RwKBatq.png)

**Comment by vinszent** [comment]
Yes these are must have features, I'll get to work on them ASAP. I'm planning on adding in 3rd party features like from BTTV as plugins.
